### ISSUE TITLE
[Keyboard Navigation-Footer-Hide the simulator]Visual focus indicator is not visible on all buttons on footer controls. 

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2609]
**User Experience:**
Users who depends on Keyboard will feel difficulty to access any controls if Focus indicator is not visible on that.

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the footer section and select 'Hide the simulator' control and select it.
4-Verify Visual focus indicator is visible or not on all footer controls on page.

**Actual Result:**
Visual focus indicator is not visible on following footer controls on page.
1.Hide the simulator
2.Download 
3.Save
4.Undo/Redo
5.Zoom in/Zoom out

**Expected Result:**
Visual focus indicator should be visible on following footer controls on page.
1.Hide the simulator
2.Download 
3.Save
4.Undo/Redo
5.Zoom in/Zoom out

**MAS Reference:**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Suggested fix:**
1.Using user interface components that are highlighted by the user agent when they receive focus
2.Using CSS to change the presentation of a user interface component when it receives focus
3.Using the default focus indicator for the platform so that high visibility default focus indicators will carry over
4.Using an author-supplied, highly visible focus indicator
5.Using script to change the background color or border of the element with focus
#div a:hover, #div a:active, #div a:focus {
background-color: #DCFFFF;
color:#66;
}
To provide an outline around the element we can use the below
a { outline: none; }

**Refer attachment file for more details:**
[MAS2.4.7_footer_visual focus indicator is not visible on footer controls.zip](https://github.com/Microsoft/pxt/files/1190521/MAS2.4.7_footer_visual.focus.indicator.is.not.visible.on.footer.controls.zip)

**Comment by veler** [comment]
@microsoftsam , according to the video in attachment of the issue, there is not outline on the buttons that have the focus. Does the color change is not enough? I'm not sure if it is very relevant.

**Comment by v-mesala** [comment]
Verified on URL: https://makecode.microbit.org/acc and issue is fixed now. Hence closing the bug
